src/libs/parse-utils.ts

```diff
@@ -0,0 +1,27 @@
+import { ParsedBody } from "src/types/meta-assistant";
+
+/**
+ * Parses Trello webhook data into a structured object
+ * 
+ * @param body JSON data from a Trello webhook payload
+ * @returns parsed Trello action details as a `ParsedBody` object
+ */
+export const parseCardDetails = (body: any) : ParsedBody => {
```

Comment:
Does it have to be any type?

Reply:
Yes, it should be better to have here just any than a long json object interface. Moreover, interface in that case should differ within files.